UOP1_Project/Assets/Scripts/Audio/AudioManager.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+
+public class AudioManager : MonoBehaviour
```

Comment:
Maybe do a [RequireComponent(AudioSource)] 

Or check if audioSource != null, and then AddComponent<AudioSource>. 

Since it's not guaranteed if someone else uses this script, they'll add an AudioSource.